packages/page/config/doctrine/Page/PageDimensionContent.orm.xml

```diff
@@ -17,5 +17,17 @@
                 <join-column name="pageUuid" referenced-column-name="uuid" on-delete="CASCADE" nullable="false"/>
             </join-columns>
         </many-to-one>
+
+        <one-to-many
+            field="navigationContexts"
+            target-entity="Sulu\Page\Domain\Model\PageDimensionContentNavigationContext"
+            mapped-by="pageDimensionContent"
+            orphan-removal="true"
+        >
+            <cascade>
+                <cascade-persist/>
+                <cascade-remove/>
```

Comment:
we should never ever use `cascade-remove` just declare on the other side `on-delete="CASCADE"` to avoid unnecessary queries on deletion of entites.